pandas/core/frame.py

```diff
@@ -2212,6 +2212,22 @@ def insert(self, loc, column, value, allow_duplicates=False):
         self._data.insert(
             loc, column, value, allow_duplicates=allow_duplicates)
 
+    def transform(self, **kwargs):
+        """
+
+        """
+        data = self.copy()
+
+        if not len(kwargs) == 1:
```

Comment:
I'm planning to add a helpful error message with how to call `.transform` here if something else is passed in. I wanted to disallow passing in multiple kwargs to avoid the issue with computations relying on each other.